### ISSUE TITLE
Not been able for other data!

### ISSUE THREAD

**Issue by asifahmad7007** [DIAGNijmegen/neural-odes-segmentation#4]
Hello, sir! I am trying the code for skin images segmentation on Colab. I download the data and make changes in dataloader as:
`class GLaSDataLoader(object):
    def __init__(self, patch_size, dataset_repeat=1, images=np.arange(0, 70), validation=False):
        self.image_fname = 'Melanoma and Non-melanoma/train_'
        self.images = images`
as my data folder is ''Melanoma and Non-melanoma''. In the data set i have named the images and respective ground truth as your tested data set. I also change the format in the data loader as:
`image = self.image_fname + index_str + '.jpg'
         mask = self.image_fname + index_str + '_anno.png'`
as my data set images are of ''jpg'' format and groundtruth ''png''
 And the first figures display cell prompts an error as:
`[Errno 2] No such file or directory: 'Warwick QU Dataset (Released 2016_07_08)/train_1.bmp'`
So the figures are blank!
I am confused. Please, sir, kindly help and guide in this regard.
Would be indebted to you!
